replication-api-impl/src/test/java/org/codice/ditto/replication/api/impl/data/ReplicatorConfigImplTest.java

```diff
@@ -90,5 +118,6 @@ public void readFromMap() {
     assertThat(config.isBidirectional(), is(true));
     assertThat(config.getFailureRetryCount(), is(5));
     assertThat(config.getVersion(), is(1));
```

Comment:
❓ Shouldn't the version be 2 in this case?

Reply:
I don't think so. The config is still version 1 and we are not converting it to a version 2

Reply:
Every time we read an old version though we are converting it to the latest version.

Reply:
fixed